index.html

```diff
@@ -67,7 +64,17 @@ <h1 id="projects">Projects</h1>
         <p></p>
         <br>
         <h1 id="contactme">Contact Me</h1>
-        <p></p>
+        <p>
+            <div class="navlinks">
```

Comment:
a/c to me social networking links(by which people can 'contact' you) should be in the 'contact section' rather than in the about one.